contrib/fmhash/fmhash.c

```diff
@@ -197,10 +195,9 @@ hash_wrapper3(struct svar *__restrict__ const sourceVal, struct svar *__restrict
 		FINALIZE;
```

Comment:
These two lines could easier (and more up to the code style) be written as
```
ABORT_FINALIZE(RS_RET_ERR);
```
but probably `RS_RET_PARAM_ERROR` would be more up to the point?